server/test/email-templates.spec.es6

```diff
@@ -3,47 +3,154 @@
+      // runs before each test in this block
+      let applicationFactory = noncommercialPermitApplicationFactory.create();
+      noncommController.translateFromClientToDatabase(applicationFactory, application);
+      application.forestName = forestName;
+    });
 
-  it('should send an email when temp outfitter application is rejected', () => {
-    const application = tempOutfitterPermitApplicationFactory.create();
-    application.status = 'Cancelled';
-    application.type = 'temp-outfitter';
-    emails.tempOutfitterApplicationRejected(application);
-    expect(true).to.be.true;
-  });
+    it('should build an object of email content for noncommercial app submission to user', () => {
+      const emailContent = emails.noncommercialApplicationSubmittedConfirmation(application);
+      const specialUseSubject = 'Your Noncommercial Open Forest permit application has been submitted for review!';
+      expect(emailContent.subject).to.be.eq(specialUseSubject);
+      expect(emailContent).to.have.all.keys('to','subject', 'body', 'html');
+      
+      expect(emailContent.body.trim()).to.include('NOT APPROVED');
+      expect(emailContent.body.trim()).to.include('Title: Special use administrator');
+      expect(emailContent.body.trim()).to.include('Start date: 12/12/2018 08:00 a.m.');
+      expect(emailContent.body.trim()).to.include('End date: 12/12/2018 04:00 p.m.');
+      expect(emailContent.html.trim()).to.include('<strong>NOT APPROVED</strong>');
+      expect(emailContent.html.trim()).to.include('<td class="border-bottom">Special use administrator</td>');
+      expect(emailContent.html.trim()).to.include('12/12/2018 08:00 a.m.');
+      expect(emailContent.html.trim()).to.include('12/12/2018 04:00 p.m.');
+    });
+
+    it('should build an object of email content for noncommercial app submission to admin', () => {
+      const emailContent = emails.noncommercialApplicationSubmittedAdminConfirmation(application);
+      expect(emailContent.subject).to.be.eq(adminSubject);
+      expect(emailContent).to.have.all.keys('to', 'subject', 'body', 'html');
+    });
+
+    it('should build an object of email content for noncommercial app acceptance to SUDS', () => {
+      const emailContent = emails.noncommercialApplicationAccepted(application);
+      expect(emailContent.subject).to.be.eq(specialUseSubject);
+      expect(emailContent).to.have.all.keys('to', 'subject', 'body', 'html');
+    });
 
-  it('should send an email when temp outfitter user cancels', () => {
-    const application = tempOutfitterPermitApplicationFactory.create();
-    application.status = 'Cancelled';
-    application.type = 'temp-outfitter';
-    emails.tempOutfitterApplicationUserCancelled(application);
-    expect(true).to.be.true;
+    it('should build an object of email content for noncommercial cancellation', () => {
+      application.status = 'Cancelled';
+      const specialUseSubject = 'The Fun Party permit application to the Mt.Baker\
```

Comment:
Caught my attention so I went digging a bit.  Looks like you're using the `eslint:recommended` for your base.  👍  My first concern was there wasn't any linting because I've gotten so used to the `airbnb` config, which would have flagged this for shadowing a upper-scoped variable of the same name.  But that's not the case, so 🎉

Reply:
yeah, i get i could remove the const and probably should.